build_scripts/build_license_directory.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# PULL IN LICENSES USING NPM - LICENSE CHECKER
+npm install -g license-checker
+
+cd ../chia-blockchain-gui || 'exit" or "cd' ... || return
+
+npm ci
+
+sum=$(license-checker --summary)
+printf "%s\n" "$sum"
+
+license_list=$(license-checker --json | jq -r '.[].licenseFile' | grep -v null)
+
+# Split the license list by newline character into an array
+IFS=$'\n' read -rd '' -a licenses_array <<< "$license_list"
+
+#print the contents of the array
+printf '%s\n' "${licenses_array[@]}"
+
+for i in "${licenses_array[@]}"; do
+    dirname="licenses/$(dirname "$i" | awk -F'/' '{print $NF}')"
+    mkdir -p "$dirname"
+    echo "$dirname"
+    cp "$i" "$dirname"
+done
+
+mv licenses/ ../build_scripts/dist/daemon
+cd ../build_scripts || 'exit" or "cd' || return
```

Comment:
Having trouble understanding the quotation marks in this. What does `'exit" or "cd'` _do_?

Reply:
These are a carryover. I changed it to "exit 1".